prometheus/prometheus.yml

```diff
@@ -11,6 +11,13 @@ scrape_configs:
     scrape_interval: 5s
     static_configs:
       - targets: ['localhost:9090', 'alertmanager:9093']
+  - job_name: 'node'
+    scrape_interval: 5s
+    static_configs:
+      - targets:
+          - 'autumn.ocamllabs.io:10080'
```

Comment:
May be we should scrape this from the `production.env` config?

Reply:
Ah that is indeed a good idea! Also, `comanche` is not our machine, so we can get rid of that! 😿

Reply:
I've pushed an update to generate this from our cluster pools configuration,